llm_on_ray/inference/serve.py

```diff
@@ -147,6 +162,9 @@ def main(argv=None):
 
     ray.init(address="auto")
     deployments, model_list = get_deployed_models(args)
+    if args.debug_mode:
+        logger.debug(f"Service is running with deployments: {str(deployments)}")
+        logger.debug(f"Service is running models: {str(model_list)}")
```

Comment:
as mentioned above, just need to set logger level to control if logger.debug output anything, no need to check debug_mode everytime you need to output debug messages. Also pls limit the use of debug only when necessary.

Reply:
This part of code should not be executed when debug mode is switched off, although the behaviors of logging are the same if we remove this judgement. Anyway, removed as you mentioned above.